lighthouse-cli/test/smokehouse/test-definitions/a11y/expectations.js

```diff
@@ -312,23 +312,6 @@ const expectations = [
             ],
           },
         },
-        'form-field-multiple-labels': {
```

Comment:
It's hinted at in one of the commit messages, but to call out why this test case was removed in the PR comments as well: multiple labels on a form field no longer result in a failure as of the changes applied in https://github.com/dequelabs/axe-core/pull/1798

Reply:
hm, do we need to change our audit implementation to examine incompletes then? or just remove this audit entirely?

if I'm reading that commit correctly, it seems like this audit will never fail anymore. at which point, why have it

Reply:
That's my read of the situation too, yep.